### PR TITLE
fix typo in helm chart

### PR DESCRIPTION
**Pull request by CoolDarran**
fix indent

### CODE REVIEW

File: scripts/helm/apollo-portal/Chart.yaml [hunk 1]
```diff
@@ -2,11 +2,11 @@ apiVersion: v2
 name: apollo-portal
 description: A Helm chart for Apollo Portal
 type: application
-version: 0.1.0
+version: 0.1.1
```
Comment (nobodyiam) [inline scripts/helm/apollo-portal/Chart.yaml:5]: Please change the version to 0.2.0 to align with the version in apollo-service, I think I forgot to change this version last time.

File: scripts/helm/apollo-service/Chart.yaml [hunk 2]
```diff
@@ -2,11 +2,11 @@ apiVersion: v2
 name: apollo-service
 description: A Helm chart for Apollo Config Service and Apollo Admin Service
 type: application
-version: 0.2.0
+version: 0.2.1
```
Comment (nobodyiam) [inline scripts/helm/apollo-service/Chart.yaml:5]: Please keep the version as 0.2.0 as it is not released yet

**Review by nobodyiam — CHANGES_REQUESTED** [review]

File: scripts/helm/apollo-portal/Chart.yaml (hunk 1 above)
Comment (CoolDarran) [reply scripts/helm/apollo-portal/Chart.yaml]: modified

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**